README.md

```diff
@@ -45,43 +39,32 @@ You can preview the production build with `npm run preview`.
 
 ## Contributors ✨
 
-Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
+Thanks goes to these wonderful developers ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
 <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
 <!-- prettier-ignore-start -->
 <!-- markdownlint-disable -->
 <table>
   <tbody>
     <tr>
-      <td align="center" valign="top" width="25%"><a href="https://github.com/dutscher"><img src="https://avatars.githubusercontent.com/u/14682?v=4?s=100" width="100px;" alt="Willy Woitas"/><br /><sub><b>Willy Woitas</b></sub></a><br /><a href="https://github.com/Neofonie/a11y/commits?author=dutscher" title="Code">💻</a></td>
```

Comment:
An dieser Stelle würde ich ein  {#each...}...{/each} für die `<td>...</td>` empfehlen, und dann einen JS-Object mit bspw:
```
const contributors = [
   { name: 'Foobar', img: ..., link: ..., involved: [ 'code', ... ]},
   ...
]
```
o.ä. aufbauen.

Reply:
Laut all-contributors generiert die Tabelle automatisch, wenn man im `contributors` per Command hinzufügt.
`all-contributors add <username> <contribution>`

Im Fedium Pull Request ist auch so umgesetzt:
https://github.com/ferdium/ferdium-app/pull/1582/files#diff-cb22095514d0cab1f755ae3a7902f7d64d10199da210fbd1eb2b29eb758b38d2

Reply:
in einer readme könnte man noch markdown tables machen anstatt table td

Reply:
> Laut all-contributors generiert die Tabelle automatisch, wenn man im `contributors` per Command hinzufügt. `all-contributors add <username> <contribution>`
> 
> Im Fedium Pull Request ist auch so umgesetzt: https://github.com/ferdium/ferdium-app/pull/1582/files#diff-cb22095514d0cab1f755ae3a7902f7d64d10199da210fbd1eb2b29eb758b38d2

ich vermute, dass da noch ein package.json-Eintrag ins Projekt muss, kann das sein? und ggf. ein eintrag in der scripts-section.

Reply:
dann muss man sich die befehle nicht merken